pkg/registry/servicecatalog/instance/strategy.go

```diff
@@ -72,6 +79,11 @@ var (
 		instanceRESTStrategies,
 	}
 	_ rest.RESTUpdateStrategy = instanceStatusUpdateStrategy
+
+	instanceReferenceUpdateStrategy = instanceReferenceRESTStrategy{
+		instanceRESTStrategies,
+	}
+	_ rest.RESTUpdateStrategy = instanceReferenceUpdateStrategy
```

Comment:
Why do we need this line? some type cast check?

Reply:
yes.